pages/en/contrib/style-guide.md

```diff
@@ -578,6 +578,37 @@ describe('my class', () => {
   }
 });
 ```
+### Callback function moved to the next line
+
+The following examples show the preferred style when callback needs to move to the next line due to line length exceeding max line length defined by eslint:
+
+**Good**:
+
+```js
+it('my long test description ...',
+function(done) {
```

Comment:
Since we are using ES6, this should be probably

```
it('my long test description ...',
() => {
  ...
});
```

(Same for the other two examples below.)